packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit_webview_widget.dart

```diff
@@ -517,8 +517,9 @@ class WebKitWebViewPlatformController extends WebViewPlatformController {
     Set<String> removedJavaScriptChannels = const <String>{},
   }) async {
     webView.configuration.userContentController.removeAllUserScripts();
-    webView.configuration.userContentController
-        .removeAllScriptMessageHandlers();
+    _scriptMessageHandlers.keys.forEach(
+      webView.configuration.userContentController.removeScriptMessageHandler,
+    );
```

Comment:
`WKUserContentController.removeAllScriptMessageHandlers()` is only available 14+

Reply:
Maybe leave a TODO to use `removeAllScriptMessageHandlers` once we have a Dart-side equivalent to `@availability`?